sonar-db/src/main/java/org/sonar/db/component/ComponentMapper.java

```diff
@@ -65,9 +65,9 @@
    */
   int countComponentByOrganizationAndId(@Param("organizationUuid") String organizationUuid, @Param("componentId") long componentId);
 
-  List<ComponentDto> selectByQuery(@Param("query") ComponentQuery query, RowBounds rowBounds);
+  List<ComponentDto> selectByQuery(@Nullable @Param("organizationUuid") String organizationUuid, @Param("query") ComponentQuery query, RowBounds rowBounds);
```

Comment:
`ComponentQuery` is used to filter result based on the component's properties.
The organization is on top of that (or totally missing, ie. nullable).